scripts/dotnet.py

```diff
@@ -511,6 +511,15 @@ def remove_dotnet(architecture: str) -> str:
     '''
     rmtree(__get_directory(architecture))
 
+def shutdown_server(verbose:bool) -> None:
+    '''
+    Shutsdown the dotnet server
+    '''
+    cmdline = [
+        'dotnet', 'build-server', 'shutdown'
```

Comment:
nice, I had no idea that such command exists! 👍

Reply:
Yeah, I knew about it but had to re-look it up when I started working on this change.